.github/workflows/arm64.yml

```diff
@@ -70,7 +70,6 @@ jobs:
           curl -v --user admin.pipeline@alfresco.com:admin http://localhost:8080/alfresco/api/discovery
       - name: "Dump all Docker containers logs"
         uses: Alfresco/alfresco-build-tools/.github/actions/docker-dump-containers-logs@v1.41.0
-        if: failure() && steps.setup-env.outcome == 'failure'
```

Comment:
I don't think there's any significant advantage in having the logs always uploaded, especially if everything went well.

I definitely see room for improvement though, as things are right now if the tests fail we don't have the Docker logs to better look into the failure, which is a shame.

Please let's move the `Dump all docker container logs` step **after** the tests are run, and let's run it in case of failures where either the `setup-env` or `run-tests` step fail.

Reply:
Sure, that's better, ill change that around